ngi_pipeline/engines/piper_ngi/local_process_tracking.py

```diff
@@ -73,12 +98,7 @@ def update_charon_with_local_jobs_status(quiet=False, config=None, config_file_p
                                                                    e))
                 LOG.error(error_text)
                 if not config.get('quiet'):
-                    mail_analysis(project_name=project_name,
-                                  sample_name=sample_id,
-                                  engine_name=engine,
-                                  level="ERROR",
-                                  info_text=error_text,
-                                  workflow=workflow)
+                    mail_ngi_log_message(sample_env,"ERROR",error_text)
```

Comment:
in general in all this code the variable name is explicitly used like 'project_id=project_id' would be better if we can keep this stile